index.js

```diff
@@ -1,7 +1,11 @@
 'use strict';
+var path = require('path');
 var osHomedir = require('os-homedir');
 var home = osHomedir();
 
 module.exports = function (str) {
-	return str.replace(home, '~');
+	str += path.sep;
```

Comment:
You need to use `path.resolve(str)` on the str, otherwise you'll end up with double `//` if the input string ends in a `/`.